### PR TITLE
fix: abstract grammar

### PR DESCRIPTION
**Pull request by Jyny**

### CODE REVIEW

File: thesis/front/abstract.tex [hunk 1]
```diff
@@ -32,43 +32,41 @@
     With the advance of science and technology,
 the computing power of smart devices has increased,
 and the size getting more compact,
-these make people easier to record sounds.
-However, as the recording equipment becomes easier to carry with,
-it also makes eavesdropping becomes more difficult to prevent.
+these make peoples easier to record sounds.
```
Comment (SCWhite) [inline thesis/front/abstract.tex:35]: I don't think we should use "peoples" in this sentence

File: thesis/front/abstract.tex [hunk 2]
```diff
@@ -32,43 +32,41 @@
...
+However, as the recording equipment becomes easier to carry,
+eavesdropping becomes more challenging to prevent.
 If a private conversation is illegally recorded or captured,
-personal privacy may be seriously violated.
+it may be seriously violating our privacy.
```
Comment (SCWhite) [inline thesis/front/abstract.tex:39]: maybe change to "our privacy might be violated seriously" ?

File: thesis/front/abstract.tex (hunk 1 above)
Comment (Jyny) [reply thesis/front/abstract.tex:35]: any advising subjects?

File: thesis/front/abstract.tex [hunk 3]
```diff
@@ -32,43 +32,41 @@
...
-this can solve the problem of eavesdropping or unauthorized recording,
-but it also affect all microphones in the same area. For example,
+this is equivalent to a denial of service (DoS) attack on the microphone's recording. The impact on the recording result is devastating.
+Suppose deploy an ultrasonic jammer in the chamber of private conversation. In that case,
+this can solve eavesdropping or unauthorized recording,
+but it also affects all microphones in the same area. For example,
 if you want to record a secret meeting as evidence,
 or you want to create a verbatim script based on the recording after the meeting.
-Thus it's unable to achieve because the interference of ultrasonic microphone jammer.
+Thus it's unable to achieve because of the interference of the ultrasonic microphone jammer.
```
Comment (SCWhite) [inline thesis/front/abstract.tex:57]: weird, may need to reconstruct this sentence

File: thesis/front/abstract.tex (hunk 2 above)
Comment (Jyny) [reply thesis/front/abstract.tex:39]: 截圖 2021-07-26 13 58 58

sounds like grammarly more prefer active voice.

File: thesis/front/abstract.tex (hunk 1 above)
Comment (SCWhite) [reply thesis/front/abstract.tex:35]: just use "people" will be enough

Comment (SCWhite) [reply thesis/front/abstract.tex:35]: or we can just use "user" instead?

File: thesis/front/abstract.tex [hunk 4]
```diff
@@ -32,43 +32,41 @@
...
 
-    Therefore, we proposes a jamming/de-jamming method
+    Therefore, we propose a jamming/de-jamming method
 based on microphone jammer and Active Noise Cancellation (ANC) technology.
-Utilize the non-linear impulse response of ultrasonic to jamming microphone,
-and de-jammed by active noise cancelling the high-correlation noise.
+Utilize the non-linear impulse response of ultrasonic to a jamming microphone
```
Comment (SCWhite) [inline thesis/front/abstract.tex:61]: marked, wait for discussion

File: thesis/front/abstract.tex [hunk 5]
```diff
@@ -32,43 +32,41 @@
...
+and de-jammed by active noise canceling the high-correlation noise.
 At the same time, we developed a sound record accessing protocol
 base on the confidentiality and non-repudiation of the cryptographic.
-Through the combination of the above technologies,
-these solve the problem that acoustic records were corrupted and not accessible
-after private conversation in area where microphone jammers are deployed.
+The above technologies
+solve the problem that acoustic records were corrupted and not accessible
+after the private conversation in the area where deployed microphone jammers.
```
Comment (SCWhite) [inline thesis/front/abstract.tex:67]: marked, need fix

File: thesis/front/abstract.tex (hunk 1 above)
Comment (Jyny) [reply thesis/front/abstract.tex:35]: `people`

File: thesis/front/abstract.tex (hunk 2 above)
Comment (Jyny) [reply thesis/front/abstract.tex:39]: keep origin

File: thesis/front/abstract.tex (hunk 3 above)
Comment (Jyny) [reply thesis/front/abstract.tex:57]: keep origin

File: thesis/front/abstract.tex (hunk 5 above)
Comment (Jyny) [reply thesis/front/abstract.tex:67]: revert line 66(65), keep origin

**Review by SCWhite — APPROVED** [review]
OK, LGTM

**Pull request merged**